src/components/Dialogs/Dialog.style.tsx

```diff
@@ -58,4 +63,15 @@ export const DialogContent = styled(MuiDialogContent)`
 export const StyledIconButton = styled(IconButton)`
   padding: 0;
   margin-left: 0.75rem;
+  display: flex;
+  align-self: flex-start;
+`;
```

Comment:
Consider disabling the ripple effect and adding the focus styling (outline) that we use in other buttons

Reply:
disabled ripple. the blue focus outline should be rendering

![Screen Shot 2021-06-07 at 8 10 27 PM](https://user-images.githubusercontent.com/44076375/121102829-91e6a600-c7cc-11eb-9ba3-fb02808acb64.png)